org.eclipse.january/src/org/eclipse/january/dataset/SliceND.java

```diff
@@ -29,8 +29,8 @@
 	private boolean expanded;
 
 	/**
-	 * Construct ND slice for whole of shape
-	 * @param shape
+	 * Constructs a SliceND Object for whole of shape
+	 * @param shape Array of shapes
```

Comment:
should be `@param shape the shape of the dataset, see {@link ILazyDataset#getShape()}`